projects/storefrontstyles/scss/products/_details.scss

```diff
@@ -0,0 +1,141 @@
+$useTabsOnPDP: false !default;
+
+%pdpTabs {
+  > div {
+    background: var(--cx-background-color, var(--cx-background));
+    &:not(.active) {
+      display: none;
+    }
+    .container > h2 {
+      display: none;
+    }
+  }
+  > h3 {
+    margin-bottom: 0;
+    &.active,
+    &:hover {
+      color: var(--cx-color, var(--cx-primary));
+    }
+    &:not(.active) {
+      cursor: pointer;
+    }
+
+    // border effect
+    &:after {
+      content: '';
+      display: block;
+      margin: 15px auto auto auto;
+      height: var(--cx-height, 5px);
+      background: var(--cx-color, var(--cx-primary));
+
+      // the tab hover effect uses a border in the :after pseudo
+      // that is animated from 0% to 100% width
+      width: 0;
+      opacity: var(--cx-opacity, 0);
+      transition: all var(--cx-transition-time, 0.6s);
+    }
+
+    &.active:after,
+    &:hover:after {
+      width: 100%;
+    }
+    &.active:after,
+    &.active:hover:after {
+      opacity: var(--cx-opacity, 1);
+    }
+    &:not(.active):hover:after {
+      opacity: var(--cx-opacity, 0.5);
+    }
+  }
+
+  @include media-breakpoint-down(sm) {
+    > h3 {
+      text-indent: 20px;
+      border-top: var(--cx-border-width, 1px)
+        var(--cx-border-color, var(--cx-light)) var(--cx-border-style, solid);
+      font-size: 18px;
+      padding: 15px 0 0 0;
+      &:before {
+        content: '+';
+        margin: 5px 15px;
+        float: right;
+        font-size: 33px;
+        bottom: 12px;
+        position: relative;
+        color: var(--cx-color, var(--cx-text));
+      }
+      &.active {
+        &:before {
+          content: '–';
+        }
+      }
+    }
+    > div {
+      padding: 15px 5px;
+    }
+  }
+
+  @include media-breakpoint-up(md) {
+    display: grid;
+    grid-template-columns: auto repeat(4, 14vw) auto;
```

Comment:
There is a bug on Ipad because of this ~1100 px width where Product Details stacks and the other lines look weird on hover and when active. I tried with this `repeat(4, minmax(14vw, auto)` it works but when you select something else than product details it resizes weirdly.

Reply:
After a 2nd look, i've moved the tabs to the left, removed the additional padding on the tab content, and ensured we always have the tab title on a single line. Looks much better now I believe. Code is a bit more complex.